app/src/main/java/com/dcrandroid/fragments/OverviewFragment.kt

```diff
@@ -154,7 +182,33 @@ class OverviewFragment : BaseFragment(), ViewTreeObserver.OnScrollChangedListene
         mixer_status_rv.adapter = MixerStatusAdapter()
         setMixerStatus()
 
-        fetchExchangeRate()
+        ivConcealReveal.setOnClickListener {
+            if (isBalanceHidden) {
+                multiWallet!!.setBoolConfigValueForKey(Constants.SHOW_HIDE_BALANCE, false)
+                isBalanceHidden = false
+                loadBalance()
+            } else {
+                multiWallet!!.setBoolConfigValueForKey(Constants.SHOW_HIDE_BALANCE, true)
+                isBalanceHidden = true
+                loadBalance()
+            }
```

Comment:
You can replace with 
```suggestion
            isBalanceHidden = !isBalanceHidden
            multiWallet?.setBoolConfigValueForKey(Constants.SHOW_HIDE_BALANCE, isBalanceHidden)
            loadBalance()
```